searches/exponential_search

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
This file is missing the `.php` file extension.  Also, please use common file naming conventions - I prefer pascal casing or camel casing.  Since most of your other files use camel case, please stick with that convention.  But if it's a PHP class file, then use the same name as the class for the file name (following autoloader convenctions).